examples/asr/speech_to_text.py

```diff
@@ -64,7 +65,10 @@
 
 @hydra.main(config_path="conf", config_name="config")
 def main(cfg):
+    # omegaconf merg trainer stuff to optim - this is necessary to be able to correctly setup LR scheduler
     trainer = pl.Trainer(**cfg.pl.trainer)
+    exp_manager(trainer, name=cfg.get("name", "ASR"), files_to_copy=[__file__])
```

Comment:
Is `name` here the equivalent of the `experiment_name` from old nemo?

Reply:
Also, could you refactor setup of exp_manager to how we do it below - `exp_manager(cfg=cfg, trainer=trainer)` and parse cfg for logger arguments like name above.

Reply:
can we add exp_manager to the .yaml and put paths to logs and checkpoints there?
```yaml
exp_manager:
  exp_name: null # here we could even infer some values from the config to auto generate the exp name
  workdir: ./output
```

Reply:
It's somewhat similar to experiment_name from the asr nemo files.
The new behaviour is to create an logging directory at f"{root_dir}/{name}/version" where version is a datetime string or an integer depending on single vs multi-node runs

Reply:
So "name" is now experiment_name, and root_path is an optional dir. Hmm, I suppose that's fine, but I quite liked the old Nemo that we could provide a whole dir + name for experiment and it would create the intermediate directories from the pwd, now the default rootdir is ~/Nemo_Experiments, which is gonna get cluttered fast without me overriding everytime. But that's not much of an ssue.

Reply:
Started a poll in slack for default directory. Please vote there